arkouda/pdarrayclass.py

```diff
@@ -578,13 +578,13 @@ def max(self) -> Union[np.float64,np.int64]:
 
     def argmin(self) -> np.int64:
         """
-        Return the max of the first minimum value of the array.
+        Return the index of the minimum value of the array.
         """
         return argmin(self)
 
     def argmax(self) -> np.int64:
         """
-        Return the index of the first maximum value of the array.
+        Return the index of the maximum value of the array.
```

Comment:
Similarly, this is the *first* maximum value.

Reply:
updated